torch/_prims/nvfuser_prims.py

```diff
@@ -0,0 +1,301 @@
+# Module for defining "primitive" operations executable by the nvFuser. This
+# list exists to decouple main set of primitives from the ones that provide a
+# lowering of the op to nvFuser’s Python interface. Mostly torch.ops.nvprims is
+# a subset of the primitives in torch.ops.prims, but some additional primitives
+# can be added in the future for the corresponding higher-level torch/aten
+# functions.
+
+from typing import Any, Dict
+
+import torch
+
+from torch._prims_common import (
+    DimsSequenceType,
+    getnvFuserDtype,
+    ShapeType,
+    TensorLikeType,
+)
+
+from torch.utils._pytree import tree_flatten, tree_unflatten
+
+nvprim_namespace = "nvprims"
+nvprim = torch.library.Library(nvprim_namespace, "DEF")
+nvprim_impl = torch.library.Library(
+    nvprim_namespace, "IMPL", "CompositeExplicitAutograd"
+)
+nvprim_autograd_impl = torch.library.Library(nvprim_namespace, "IMPL", "Autograd")
+nvprim_meta_impl = torch.library.Library(nvprim_namespace, "IMPL", "Meta")
+
+nvprim_names = [
+    "abs",
+    "acos",
+    "asin",
+    "atan",
+    "atanh",
+    "cos",
+    "cosh",
+    "bitwise_not",
+    "ceil",
+    "erf",
+    "erfc",
+    "exp",
+    "expm1",
+    "floor",
+    "imag",
+    "isfinite",
+    "lgamma",
+    "log",
+    "log1p",
+    "log2",
+    "log10",
+    "real",
+    "reciprocal",
+    "neg",
+    "round",
+    "rsqrt",
+    "sin",
+    "sinh",
+    "sqrt",
+    "tan",
+    "tanh",
+    "trunc",
+    "add",
+    "atan2",
+    "bitwise_and",
+    "bitwise_or",
+    "bitwise_xor",
+    "div",
+    "eq",
+    "fmod",
+    "ge",
+    "gt",
+    "le",
+    "lt",
+    "mul",
+    "ne",
+    "pow",
+    "remainder",
+    "sub",
+    "broadcast_in_dim",
+    "where",
+    "convert_element_type",
+    "sum",
+    "var",
+    "amax",
+    "amin",
+]
+
+
+def _autograd_fn(prim):
```

Comment:
nitpick: this is copied from `torch/_prims/__init__.py. should we refactor it and reuse the function instead of making a copy?

Reply:
Sure, will do that.